### PR TITLE
don't select "yes" on return

### PR DESCRIPTION
**Pull request by pazz**
in order to prevent accidental buffer-closes (cf. issue #875)

### CODE REVIEW

File: alot/commands/globals.py [hunk 1]
```diff
@@ -457,7 +455,7 @@ def one_buffer(prompt=True):
         if (isinstance(self.buffer, buffers.EnvelopeBuffer) and
                 not self.buffer.envelope.sent_time):
             if (not self.force and (yield ui.choice('close without sending?',
-                                                    select='yes', cancel='no',
+                                                    select=None, cancel='no',
```
Comment (lucc) [inline alot/commands/globals.py]: `None` is the default anyway.

File: alot/commands/globals.py [hunk 2]
```diff
@@ -78,8 +77,7 @@ def apply(self, ui):
 
         if ui.db_was_locked:
             msg = 'Database locked. Exit without saving?'
-            if (yield ui.choice(msg, select='yes', cancel='no',
-                                msg_position='left')) == 'no':
+            if (yield ui.choice(msg, select='yes', cancel='no')) == 'no':
```
Comment (lucc) [inline alot/commands/globals.py]: Why did you change the `msg_position`? Is it related to the select stuff?

And why do you still `select='yes'` here?

What do I need to do in order to get to this prompt (to test it)?

**Comment by pazz** [comment]
Quoting Lucas Hoffmann (2017-06-04 08:10:32)
> @lucc requested changes on this pull request.
> 
> ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
> 
> In alot/commands/globals.py:
> 
> > @@ -457,7 +455,7 @@ def one_buffer(prompt=True):
>          if (isinstance(self.buffer, buffers.EnvelopeBuffer) and
>                  not self.buffer.envelope.sent_time):
>              if (not self.force and (yield ui.choice('close without sending?',
> -                                                    select='yes', cancel='no',
> +                                                    select=None, cancel='no',
> 
> None is the default anyway.

yea, true

**Comment by pazz** [comment]
ARGS! how do i amend a PR? I used "git commit --amend ." locally to fix these issues you mentioned, and could only push the branch again after deleting it upstream, which means that the PR closes and cannot be reopened.  #1059

**Comment by lucc** [comment]
`git push -f`?

**Comment by pazz** [comment]
Quoting Lucas Hoffmann (2017-06-05 23:18:15)
> git push -f?

I think i tried that.. but thanks, i'll double check next time. Sorry
for the mess.
